assets/css/detours/_detour_map.scss

```diff
@@ -12,6 +12,13 @@
   }
 }
 
+.c-detour_map--original-route-shape-diverted {
+  stroke: $black;
```

Comment:
I think I may have gotten this from the wrong part of the Figma.

Reply:
... so I think I need to update this to match the designs in the component section of the Figma, rather than on the map itself.

Reply:
[this should fix that](https://github.com/mbta/skate/pull/2455/commits/cd0448cee0609a8b08d9e85a9e108c297f660b73)

Reply:
Allllmost! [this](https://github.com/mbta/skate/pull/2455/commits/53b2368e0e5f149e9588dc21f18e7ca9773a2234) should be the rest of the fix

Reply:
Okay [there we go](https://github.com/mbta/skate/pull/2455/commits/62e4d0f86061344d7695b0fca0f5a7c1ba731f3f)